app/src/main/java/org/mozilla/focus/experiments/NimbusSetup.kt

```diff
@@ -82,7 +84,16 @@ fun createNimbus(context: Context, url: String?): NimbusApi {
             }
 
             applyPendingExperiments()
+            NimbusExperiments.nimbusInitialFetch.start()
             fetchExperiments()
+
+            register(object : NimbusInterface.Observer {
+                override fun onExperimentsFetched() {
+                    NimbusExperiments.nimbusInitialFetch.stop()
```

Comment:
Just to clarify, based on the description of https://github.com/mozilla-mobile/focus-android/issues/6847, should we be recording the event when we are not able to fetch the experiments and going to the default experience ? :)

Reply:
Maybe there is a metric that I'm ignoring, please help me to clarify, I think we will need to also have amount of users that try to download. Having the total users that try to download and the users download successfully, I think we could figure out how many failed to download.

Reply:
Base on the documentation  for `cancel`:
> Abort a previous start call. No error is recorded if no start was called.

Reply:
It's worth asking **why** we're collecting this information.

We're collecting this to make a decision on how long to set the timeout. The fetch will continue even if the splash screen is dismissed (and by the looks of it here: on every startup).

I don't **think** we need to collect whether or not the initial fetch was completed in time for the onboarding.

What do you think?